content/guides/models/track/log/distributed-training.md

```diff
@@ -45,22 +51,24 @@ if __name__ == "__main__":
         train(args)
 ```
 
-Explore the W&B App UI to view an [example dashboard](https://wandb.ai/ayush-thakur/DDP/runs/1s56u3hc/system) of metrics tracked from a single process. The dashboard displays system metrics such as temperature and utilization, that were tracked for both GPUs.
+Explore this example dashboard in the W&B App UI to [view metrics tracked from a single process](https://wandb.ai/ayush-thakur/DDP/runs/1s56u3hc/system). The dashboard displays system metrics for both GPUs, such as temperature and utilization.
 
-{{< img src="/images/track/distributed_training_method1.png" alt="" >}}
+{{< img src="/images/track/distributed_training_method1.png" alt="Workspace that shows four line plot panels. From left to right the plots are: GPU Utilization (%), GPU Temp (C), GPU Time Spent Accessing Memory (%), and GPU Memory Allocated (%)" >}}
 
 However, the loss values as a function epoch and batch size were only logged from a single GPU.
 
-{{< img src="/images/experiments/loss_function_single_gpu.png" alt="" >}}
+{{< img src="/images/experiments/loss_function_single_gpu.png" alt="Two line plot panels in an example workspace. The left plot shows the loss measured as a function of step. The right plot show the loss recorded at each batch." >}}
 
-### Method 2: Many processes
+## Track multiple processes
 
```

Comment:
Ideally you'd have a sentence here explaining what the subsections are about, or you would skip this H2 and promote the ones below from H3 to H2. Skimming the structure of the rest of the page, I bet the promotion option would be fine.